public/javascripts/controllers/buyer-ctrl.js

```diff
@@ -199,6 +202,59 @@ angular.module('jasmic.controllers')
                 $scope.demand.de_until = moment().add(7, 'days').toDate();
             };
 
+            $scope.editDemandForm = function (index) {
```

Comment:
@nick22891 Whilst we don't always do it, please remember to include function comments to describe what a function does.